CometServer/Entities/Universe.cpp

```diff
@@ -167,18 +167,19 @@ namespace entity
 				{
 					if (p_entity1 == p_entity2) continue;
 					//Test for collision.
-					#ifndef NO_COLLISION
+					/*#ifndef NO_COLLISION
 					AbstractCollisionShape& shape1 = *p_entity1->shape;
 					AbstractCollisionShape& shape2 = *p_entity2->shape;
 					shape1.InviteForCollision(shape1.GetBoundingBox(), shape2.GetBoundingBox(), shape2);
-					#endif
+					#endif*/
 				}
 			}
 		}
 	}
 
 	void Universe::TestVisibility() //TODO: Maybe call this TestVision?
 	{
+		vision_partitioner.Reset();
```

Comment:
This was quite a serious bug, without this reset, the size of the vision partition kept growing with every simulation frame.